equinix/resource_metal_virtual_circuit.go

```diff
@@ -269,7 +269,7 @@ func getVCStateWaiter(client *packngo.Client, id string, timeout time.Duration,
 			vc, _, err := client.VirtualCircuits.Get(
 				id,
 				&packngo.GetOptions{Includes: []string{
-					"project", "port", "virtual_network",
+					"project", "virtual_network",
 					"vrf",
 				}}, // TODO: we are not using the returned VC. Remove the includes?
```

Comment:
The TODO wondered about this. Do you think the other Includes could be removed?

Reply:
Yeah, they might as well be removed.  I left them in order to keep the change as targeted as possible, but there's no reason to keep kicking this can down the road.